notebooks/tabular_examples/model_agnostic/Simple California Demo.ipynb

```diff
@@ -0,0 +1,293 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "partition_tree = shap.utils.partition_tree(X)\n",
+    "plt.figure(figsize=(15, 6))\n",
+    "sp.cluster.hierarchy.dendrogram(partition_tree, labels=X.columns)\n",
+    "plt.title(\"Hierarchical Clustering Dendrogram\")\n",
+    "plt.xlabel(\"feature\")\n",
+    "plt.ylabel(\"distance\")\n",
+    "plt.show()"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2023-10-14T10:50:51.556775120Z",
+     "start_time": "2023-10-14T10:50:51.358066248Z"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Explain the instance"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "outputs": [],
+   "source": [
+    "# build a masker from partition tree\n",
+    "masker = shap.maskers.Partition(X, clustering=partition_tree)\n",
+    "\n",
+    "# build explainer objects\n",
+    "raw_explainer = shap.PartitionExplainer(model.predict, X)\n",
+    "masker_explainer = shap.explainers.Partition(model.predict, masker)\n",
```

Comment:
Let's standardize and just use `shap.PartitionExplainer` here for both explainers.